### ISSUE TITLE
Accessing specific elements in custom loss function?

### ISSUE THREAD

**Issue by dhrushilbadani** [keras-team/keras#13156]
When defining a custom loss function for a classification problem, is there a way to access particular elements of y_true and y_pred?

Use-case: multi-label classification problem where I wanna penalize the model extra if I predict a false positive for class 5 i.e. `y_true[5] == 0` but  `y_pred[5] == 1`

I'm defining the loss like:
```
def loss(y_true, y_pred):
      wt = 10 if (y_true[5]==0 and y_pred[5]==1) else 1
      return wt * binary_crossentropy(y_true, y_pred)
```

My batch size is > 1 (256) and i'm using `fit_generator` - if that makes any difference. Thanks!
